apps/dashboard/src/middleware.ts

```diff
@@ -93,6 +94,18 @@ export async function middleware(request: NextRequest) {
       return rewrite(request, `/published-contract${pathname}`);
     }
   }
+
+  // redirect /team/~/... to /team/<first_team_slug>/...
```

Comment:
why can't this just be `/team`? why do we need the extra `/~` in there?

Reply:
If we want to have  /team/<otherSlugs> instead of /team/~/<otherSlugs>

* That breaks the usual format - it should be /team/<team-slug>/<...otherSlugs>
* We'll have to determine if otherSlugs[0] is not a valid team slug and then insert the first team's slug before them

Reply:
I see you want to allow the deep-linking to something *inside* the default team - okay